torch/csrc/autograd/python_function.cpp

```diff
@@ -104,7 +109,7 @@ auto PyNode::apply(variable_list&& inputs) -> variable_list {
   // Massage a C++ variable_list into a Python arguments tuple
   auto num_inputs = inputs.size();
   THPObjectPtr pyInputs(PyTuple_New(num_inputs));
-  if (!pyInputs) throw python_error();
+  if (!pyInputs) throw_python_error();
```

Comment:
Looks like there are many more `throw python_error()` in this file. Should we replace those too?

Reply:
I only applied this change to PyNode, all other places of `throw python_error()`, they are actually functions that is used by the PyNode, so PyNode will catch those exceptions and persist them as well. We don't need to change the error throwing logic outside of the PyNode.